xcp_d/tests/test_interfaces_censoring.py

```diff
@@ -25,7 +27,7 @@ def test_generate_confounds(fmriprep_with_freesurfer_data, tmp_path_factory):
     df.to_csv(confounds_tsv, sep="\t", index=False, header=True)
 
     # Run workflow
-    interface = GenerateConfounds(
+    interface = censoring.GenerateConfounds(
```

Comment:
is this for readability?

Reply:
Yes. I like to import the module for these module-level test files, so I don't have to import the individual classes and functions that will be tested.